keyboards/melgeek/z70ultra_v1/info.json

```diff
@@ -0,0 +1,391 @@
+{
+    "keyboard_name": "MelGeek Z70Ultra v1",
+    "manufacturer": "MelGeek",
+    "url": "",
+    "maintainer": "melgeek001365",
+    "usb": {
+        "vid": "0xEDED",
+        "pid": "0x6575",
+        "device_version": "0.0.1"
+    },
+	"processor": "STM32F303",
+    "bootloader": "stm32-dfu",
+    "board": "QMK_PROTON_C",
```

Comment:
Is this actually using a proton C? 

Given availability of them, I'm assuming no.   And if that is the case, should avoid using the board files for it, and use the default board files (and enable whichever features you need)